### DISCUSSION TITLE
RDTClient.web.exe - Not included in package

### DISCUSSION THREAD

**Discussion by MrJoshGit** [rogerfar/rdt-client#474] [General]
Trying to install this on windows.  I am trying to locate this file which I thnk should be in the web folder.  However latest release does not have this file included.  Where can I get this file?

**Comment by MrJoshGit** [comment]
Nevermind  I figured it out.  I have to download the release not the source code.

**Reply by RiverdaleBoy** [reply]
I have the same problem and am not that techie. How do I downoad the release and not the source code

**Reply by RiverdaleBoy** [reply]
Figured it out as well. On Github, on the far right, there is a link to Releases. From there I was able to get the .exe file. Who knew? You all did and now I know!

**Comment by rogerfar** [comment]
Indeed, or you can use the Docker container too.
